Umbraco-Cloud/Set-Up/Application-Insights/index.md

```diff
@@ -0,0 +1,35 @@
+# Application insigths on Umbraco cloud
```

Comment:
Hey @mclausen 

Awesome job on this article, one thing I notice is the inconsistency in Application Insights, sometimes the first letters are capitalized, sometimes not, could you make it consistent throughout this page? 🙂

And also Umbraco Cloud has the same issue in this sentence 😉